geocity/apps/submissions/forms.py

```diff
@@ -1424,6 +1480,12 @@ def __init__(self, request, submission, *args, **kwargs):
         ).all()
         self.fields["authorised_departments"].label = _("Département autorisé")
 
+        # TOFIX: reports that are linked to transaction should not
+        # be able to be generated here but rather through the transactions' tab
+        # For now, we have to get the last transaction, in order for the reports
+        # linked payments to work.
```

Comment:
Would be nice to create a user story to not forget that

Reply:
Done